### ISSUE TITLE
Setup not starting

### ISSUE THREAD

**Issue by cljnilsson** [Willqvist/MapleNode#1]
I was working on the same project myself and figured I could just use this as a dependency instead.

However, I cannot test the project.

Steps:
1. Download .zip file
2. extract folder
3. npm install
4. modify database.MN
5. node app
6. get errors
https://i.gyazo.com/c38593bf7e39873fede250da54fc9147.png

The SQL table generation does not run for some reason

**Comment by Willqvist** [comment]
Sorry for 6 month late response... . Did you delete the setup.MN? If not, delete it otherwise the server thinks the setup is complete and that may cause that error.

**Comment by cljnilsson** [comment]
Right so I came back to this quite a bit later.

I did not get to get the demo working. However I specifically only wanted the character render part.
This required some edits on my part but I think I got it to work with limited testing.

In my humble opinion I threw away the existing SQL file and made a cleaner one so I do not need to inject a sql instance.

Secondly threw in some async/await keywords for cleaner code.

However, recent developments have most character renders animated which would be a pretty cool feature if you ever intend to develop this in the future.

Regardless, huge thanks for making this. Saved me a headache or two.
